server/urls.py

```diff
@@ -26,8 +28,21 @@
     path('admin/', admin.site.urls),
 
     url(r'^$', server.views.index, name='index'),
+]
+
+if settings.SITE_URL == 'https://iodide.io/':
```

Comment:
Would this be extremely-alpha.iodide.io?

How about defining an environment variable with the following format:

`<pathspec>=<redirect>;<pathspec>=<redirect>;...`

in this case:

`dist/.*=https://iodide-project.github.io/dist/;docs/.*=https://iodide-project.github.io/docs/;pyodide-demo/.*=https://iodide-project.github.io/pyodide-demo/`

This is less iodide.io specific, and would allow us to configure/tweak the exact behaviour inside the heroku environment.

Reply:
> Would this be extremely-alpha.iodide.io?

This came out of a discussion of moving what's currently at `extremely-alpha.iodide.io` to `iodide.io` without blitzing what's currently at `iodide.io`.

> How about defining an environment variable with the following format.

Sure, that's easy enough.  I'll update the PR.

Reply:
Oh, that's a good idea.  But I'm not sure how we would implement it.  Currently Github pages handles `iodide.io`, and I don't think there's a way to set up dynamic redirects there.

But you're right -- this whole PR is on the assumption that `iodide.io` would eventually point to our server on Heroku, which may not have been the right assumption.